source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst

```diff
@@ -50,8 +55,31 @@ is supported on platforms:
 #. Power on the LVDS board
 #. Run the following command:
 
-	.. code-block:: bash
+	.. code-block:: console
 
 		root@am62xx-evm:~# modprobe ili210x
 
 After re-initializing the module, the touch IC should now be registered as an input and touch should be functional.
+
+B/B-P1 revision
```

Comment:
```suggestion
AM62B SK & AM62B-P1
```

Better to have consistency across variants names - https://github.com/TexasInstruments/processor-sdk-doc/pull/105/files#diff-7eb9b090bbfd51a34a333ebfdd6d81739b90943f430fc2aad0185f8596915d00R10

Reply:
Done